engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -176,15 +180,21 @@ public WorldRendererImpl(Context context, GLBufferPool bufferPool) {
     }
 
     private void initRenderingSupport() {
-        frameBuffersManager = new FrameBuffersManager();
+        dynamicFBM = new DynamicFBM();
+        staticFBM = new StaticFBM();
+        shadowMapResolutionDependentFBM = new ShadowMapResolutionDependentFBM();
 
-        context.put(FrameBuffersManager.class, frameBuffersManager);
+        context.put(DynamicFBM.class, dynamicFBM);
+        context.put(StaticFBM.class, staticFBM);
+        context.put(ShadowMapResolutionDependentFBM.class, shadowMapResolutionDependentFBM);
```

Comment:
You're right, but the same context is used for Nodes. How about getting back to this later? Can I put a TODO for this, please? 😄